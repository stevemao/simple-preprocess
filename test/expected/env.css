body {
  background: red;
}
body {
  background: yellow;
}
body {
  background: green;
}
