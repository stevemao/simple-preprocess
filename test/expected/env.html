<script charset="utf-8">
  var value = 'prod';
</script>
<script charset="utf-8">
  var value = 'test and prod';
</script>
